OWNERS_ALIASES

```diff
@@ -1,14 +1,21 @@
 aliases:
   # business hours oncall for prow etc (https://go.k8s.io/oncall)
   test-infra-oncall:
-  - BenTheElder
+  - chaodaig
```

Comment:
This is a dup, mind delete one or the other?

Reply:
Dup with line https://github.com/kubernetes/test-infra/pull/20007/files#diff-4985b733677adf9dda6b5187397d4700868248ef646d64aecfb66c1ced575499R10

Reply:
fixed